web/yo/app/scripts/controllers/gene.js

```diff
@@ -2508,6 +2519,152 @@ angular.module('oncokbApp')
                     }
                 });
             };
+            function obsoletedUUIDs(type, mutation, tumor, TI, treatment) {
```

Comment:
This is the function to get list UUIDs about the obsoleted items? Could you write some comments for this function?